src/tests/gui/test_gui2.cpp

```diff
@@ -569,6 +571,15 @@ BOOST_AUTO_TEST_CASE(test_make_test_fake)
 
 namespace {
 
+template<>
+struct dialog_tester<addon_auth>
+{
+	addon_auth* create()
```

Comment:
Please return std::unique_ptr or std::shared_ptr instead of a raw owning pointer.

Reply:
The tests are currently set up to want a `T*`.  Changing that is outside the scope of this PR.